src/legacy/core_plugins/tile_map/public/components/wms_interna_options.tsx

```diff
@@ -0,0 +1,179 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { EuiLink, EuiSpacer, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { VisOptionsSetValue } from 'ui/vis/editors/default';
+import { TextInputOption } from '../../../kbn_vislib_vis_types/public/controls/text_input';
+
+export interface TmsLayer {
+  id: string;
+  origin: string;
+  minZoom: string;
+  maxZoom: number;
+  attribution: string;
+}
+
+interface WmsOptions {
+  enabled: boolean;
+  url: string;
+  selectedTmsLayer: TmsLayer;
+  options: {
+    attribution: string;
+    format: string;
+    layers: string;
+    styles: string;
+    version: string;
+  };
+}
+
+interface WmsInternalOptions {
+  wms: WmsOptions;
+  setValue: VisOptionsSetValue;
+}
+
+function WmsInternalOptions({ wms, setValue }: WmsInternalOptions) {
+  const wmsLink = (
+    <EuiLink href="http://www.opengeospatial.org/standards/wms" target="_blank">
+      <FormattedMessage id="tileMap.wmsOptions.wmsLinkText" defaultMessage="here" />
+    </EuiLink>
+  );
+
+  const setOptions: VisOptionsSetValue = (paramName, value) =>
+    setValue('options', {
+      ...wms.options,
+      [paramName]: value,
+    });
+
+  return (
+    <>
+      <EuiSpacer size="s" />
+      <EuiText size="s">
+        <FormattedMessage
+          id="tileMap.wmsOptions.wmsDescription"
+          defaultMessage="WMS is an OGC standard for map image services. For more information, go {wmsLink}."
+          values={{ wmsLink }}
+        />
+      </EuiText>
+      <EuiSpacer size="s" />
+
+      <TextInputOption
+        label={<FormattedMessage id="tileMap.wmsOptions.wmsUrlLabel" defaultMessage="WMS url*" />}
```

Comment:
We need to try to make these "footnotes" accessible. I'm thinking that you'll want to do some form of:

```html
<label>WMS url <sup aria-describedby="wmsInternalOptionsFootnote">*</sup></label>

<p id="wmsInternalOptionsFootnote">*If this parameter is incorrect, maps will fail to load.</p>
```

Or maybe @myasonik has some ideas as well.

Reply:
`aria-label`, `aria-labelledby` and `aria-describedby` generally don't work on non-interactive and/or non-widget role elements. 

I tested adding `role=tooltip` to various elements but that's not well supported...

So I think the best recommendation I can give is to add this text to the help text of the inputs so it's read as part of the description of the input.

To clean up the reading, I'd also hide the asterisk and the final footnote from screen readers with `aria-hidden=true`.

Reply:
Just a small note to add, also just noticed that we could pass in `describedByIds` which would similarly work to add more description text to the input but, after testing it with Daniil, we decided against it. 

Using `describedByIds` puts the repeated footnote first and the unique help text second but a user is likely to navigate away from the input before it's done reading if it sounds like we're reading the same text over and over again.

Reply:
Thanks @myasonik for your help!
The PR updated with suggested changes